cvat-core/src/ml-model.ts

```diff
@@ -30,6 +30,10 @@ export default class MLModel {
         return Array.isArray(this.serialized.labels_v2) ? [...this.serialized.labels_v2] : [];
     }
 
+    public get supportedShapeTypes(): ShapeType[] {
+        return this.serialized.supported_shape_types;
```

Comment:
In `SerializedModel` interface supported_shape_types may be undefined

So, maybe we need a default value here
If for any tracker supported_shape_types are guaranteed, maybe empty array is a good default value. 

Alternatively need to specify that supportedShapeTypes may return undefined also.

Reply:
I added `undefined`. IMO, it's better than adding a default, since it might help catch this method being used on non-trackers.